lib/Doctrine/Migrations/Tools/Console/Command/DiffCommand.php

```diff
@@ -82,7 +71,7 @@ protected function configure() : void
                 null,
                 InputOption::VALUE_OPTIONAL,
                 'Check Database Platform to the generated code.',
-                true
+                1
```

Comment:
Why `int` instead of `bool`? This command also shouldn't have a value as it's just a switch IIRC.

Reply:
phpstan complains

`Parameter #5 $default of method Symfony\Component\Console\Command\Command::addOption() expects int|string|false|null, true given.`

Reply:
Ok. That's something we should investigate separately.